l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py

```diff
@@ -216,6 +252,45 @@ def get_taxes(self):
             ('company_id', '=', self.company_id.id),
         ])
 
+    @api.multi
+    def get_ob_totals(self):
+        '''return totals of opening balance'''
+        self.env.cr.execute(
+            'select sum(l.credit), sum(l.debit), count(distinct a.id) '
+            'from account_move_line l, account_account a, '
+            '     account_account_type t '
+            'where a.user_type_id = t.id '
+            'and l.account_id = a.id '
+            'and l.date < %s '
+            'and (l.company_id=%s or l.company_id is null) '
+            'and t.include_initial_balance = true ',
+            (self.date_start, self.company_id.id, ))
+        row = self.env.cr.fetchall()[0]
+        return dict(
+            credit=round(row[0] or 0.0, 2),
+            debit=round(row[1] or 0.0, 2),
+            count=row[2] or 0
+        )
+
+    @api.multi
+    def get_ob_lines(self):
+        '''return opening balance entries'''
+        self.env.cr.execute(
+            'select a.id, a.code, sum(l.balance) '
+            'from account_move_line l, account_account a, '
+            '     account_account_type t '
+            'where a.user_type_id = t.id '
+            'and a.id = l.account_id and l.date < %s '
+            'and (l.company_id=%s or l.company_id is null) '
```

Comment:
It should technically, logically and functionally not be possible to have lines without a company ;-)

Reply:
You're right, it's all over the code though, eg. [here](https://github.com/OCA/l10n-netherlands/blob/10.0/l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py#L226)

I was lazy to correct them all, so maybe it can be an improvement for later that can then be back/forward ported to all of them.

Reply:
Yes, alright

Reply:
Forward-ported to V13 (see https://github.com/OCA/l10n-netherlands/pull/253) with removed `l.company_id is null`, as suggested.